### PR TITLE
Fix bug of converting system properties from EventData to Spring message in Eventhub binder

### PR DESCRIPTION
**Pull request by yiliuTo**
This pr is to fix the bug in #22446 , when converting a EventData to Spring message, not all system properties of EventData are transfered to Spirng msg.

### CODE REVIEW

File: sdk/spring/azure-spring-cloud-test-eventhubs/src/test/java/com/azure/spring/test/eventhubs/stream/binder/EventHubBinderSyncModeIT.java [hunk 1]
```diff
@@ -66,7 +66,7 @@
         public Consumer<Message<String>> consume() {
             return message -> {
                 LOGGER.info("EventHubBinderRecordModeIT: New message received: '{}'", message.getPayload());
-                if (message.getPayload().equals(EventHubBinderSyncModeIT.message)) {
+                if (message.getPayload().equals(EventHubBinderSyncModeIT.message) && message.getHeaders().containsKey("x-opt-enqueued-time")) {
```
Comment (chenrujun) [inline sdk/spring/azure-spring-cloud-test-eventhubs/src/test/java/com/azure/spring/test/eventhubs/stream/binder/EventHubBinderSyncModeIT.java:69]: What is this used for?

Comment (yiliuTo) [reply sdk/spring/azure-spring-cloud-test-eventhubs/src/test/java/com/azure/spring/test/eventhubs/stream/binder/EventHubBinderSyncModeIT.java:69]: To test whether headers except for those four we explicitly added are put in spring messages.

**Review by backwind1233 — APPROVED** [review]
LGTM

**Review by chenrujun — APPROVED** [review]
LGTM.

**Pull request merged**